style.css

```diff
@@ -1 +1,384 @@
 /*  Add your styling here */
+
+* {
+    font-family: 'Indie Flower', cursive;
+}
+
+body {
+    background-color: pink;
+}
+
+/* header */
+
+header {
+    border: 3px solid black;
+    margin: 5px;
+}
+
+
+.logo {
+    width: 30%;
+    height: auto;
+    border: 1px solid black;
+}
+
+/* responsive nav-bar */
+
+.menu-bar {
+    display: flex;
+    justify-content: space-around;
+    background-color: rgb(228, 112, 208);
+    padding: 5%;
+}
+
+.menu li {
+    list-style-type: none;
+    padding: 11.5%;
+    background-color: grey;
+}
+
+.menu {
+    position: fixed;
+    right: -100%;
+    transition: all.5s;
+    padding: 5.5%;
+    width: 15%;
+} 
+
+ #check {
+    display: none;
+}
+
+#check:checked ~ .menu {
+    right: 0;
+}
+
+/* anchor styling */
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+a:hover {
+    font-weight: bolder;
+    color: purple;
+}
+
+/* carousel and text section */
+
+.carousel-section {
+    display:flex;
+    flex-direction: column;
+    background-color: thistle;
+    border: 5px solid black;
+    margin: 5px;
+}
+
+.hello-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+}
+
+.hello-text h1 {
+    padding-top: 15px;
+    font-size: 25px;
+}
+
+.hello-text p {
+    padding: 1px 30px;
+    border-bottom: 1px solid black;
+    width: 60%;
+    padding: 25px;
+}
+
+.carousel {
+    display:flex;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    margin: 5%;
+    background-color: rgb(241, 224, 227);
+    border: 2px solid grey;
+    border-radius: 2%;
+    list-style-type: none;
+    height: 30%;
+    width: 70%;
+}
```

Comment:
This `carousel` is a `ul` and all `ul`s are given a default `padding-left` by the browser. Add `padding-left: 0` here to remove this undesired spacing (and do this for all your `ul` elements in the future)